doc/tutorial/Server.lhs

```diff
@@ -753,7 +753,30 @@ myHandler :: Server MyHandler
 myHandler = return $ addHeader 1797 albert
 ```
 
-Note that the type of `addHeader x` is different than the type of `x`!
+Note that the type of `addHeader x` is different than the type of `x`! More
```

Comment:
Shouldn't this say that the type of `addHeader headerValue x` is different from `x`?